cmd/generate/config/rules/gcp.go

```diff
@@ -29,8 +29,8 @@ func GCPAPIKey() *config.Rule {
 	r := config.Rule{
 		RuleID:      "gcp-api-key",
 		Description: "Uncovered a GCP API key, which could lead to unauthorized access to Google Cloud services and data breaches.",
-		Regex:       generateUniqueTokenRegex(`AIza[0-9A-Za-z\\-_]{35}`, true),
-
+		Regex:       generateUniqueTokenRegex(`(?-i:AIza)[0-9a-z_-]{35}`, true),
+		Entropy:     3.0,
```

Comment:
There's no downside to adding entropy to this rule. My tests show that keys typically have an entropy >4.